probedata.go

```diff
@@ -181,6 +182,42 @@ func (p *ProbeData) FirstAttachmentStream() *Stream {
 	return p.firstStream(StreamAttachment)
 }
 
+// CountVideoStreams returns the count of the video streams
+func (p *ProbeData) CountVideoStreams() uint32 {
+	return p.countStreams(StreamVideo)
+}
+
+// CountAudioStreams returns the count of the audio streams
+func (p *ProbeData) CountAudioStreams() uint32 {
+	return p.countStreams(StreamAudio)
+}
+
+// CountSubtitleStreams returns the count of the subtitle streams
+func (p *ProbeData) CountSubtitleStreams() uint32 {
+	return p.countStreams(StreamSubtitle)
+}
+
+// CountDataStreams returns the count of the data streams
+func (p *ProbeData) CountDataStreams() uint32 {
+	return p.countStreams(StreamData)
+}
+
+// CountAttachmentStreams returns the count of the attacment streams
+func (p *ProbeData) CountAttachmentStreams() uint32 {
+	return p.countStreams(StreamAttachment)
+}
+
+func (p *ProbeData) countStreams(streamType StreamType) uint32 {
```

Comment:
Why return uint32 instead of the normal int?

Reply:
Note: On go an int is always 32bit, even on 64bit operating systems. To get 64 bits use int64.